docker/Dockerfile-lb

```diff
@@ -1,15 +1,18 @@
-FROM gliderlabs/alpine:3.4
-
+FROM gliderlabs/alpine:3.3
```

Comment:
The version of nginx that's built for 3.4 is 1.11. This is fine but it requires a few tweaks (mainly `var/lib/nginx` has to be created manually)

To avoid unkown-unknowns, it was easier to go with 3.3 and lock nginx version to v1.8.1-r1 (same version previously used on the load balancer)